python/tk_multi_publish2/base_hooks/pre_publish.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2018 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class PrePublishHook(HookBaseClass):
```

Comment:
For your simple hook you don't need to go through the whole `base_class` setup. You can skip this and leave out the base class during instantiation.  (see other comment)